### DISCUSSION TITLE
Move between open editors tabs

### DISCUSSION THREAD

**Discussion by NextPage66** [codefori/vscode-ibmi#2179] [Q&A] [answered]
I typically have multiple source files open for research and maintenance.  They do not all fit in the header bar so many of them are off screen. 
 Is there a way to move to a tab that is not displayed without going to the open editors explorer?

**Comment by sebjulliand** [comment, marked as answer]
There is a small scrollbar underneath the tabs that appears when there are too many open to fit. You can use your mouse-wheel to scroll the tabs, too.

**Reply by NextPage66** [reply]
Wow.  I didn't even see that scroll bar.   It's so small.  And the tab scroll is awesome.  Thanks!

**Reply by chrjorgensen** [reply]
There's a lot of settings in VS Code for how the tabs should behave, if you're not satisfied with the current setup.
